docs/sources/release-notes/v2.5.md

```diff
@@ -0,0 +1,36 @@
+---
+title: "Grafana Mimir version 2.5 release notes"
+menuTitle: "V2.5 release notes"
+description: "Release notes for Grafana Mimir version 2.5"
+weight: 300
+---
+
+# Grafana Mimir version 2.5 release notes
+
+Grafana Labs is excited to announce version 2.5 of Grafana Mimir.
+
+The highlights that follow include the top features, enhancements, and bugfixes in this release. For the complete list of changes, see the [changelog](https://github.com/grafana/mimir/blob/main/CHANGELOG.md).
+
+## Features and enhancements
+
+- **Alertmanager Discord support**
+  The alertmanager can now be configured to send alerts into Discord channels.
+
+- **Configurable TLS minimum version and cipher suites**
+  We added the two new flags `-server.tls-min-version` and `-server.tls-cipher-suites`, they can be used to define the minimum version and the supported cipher suites in all HTTP and gRPC servers in Mimir.
+
+- **Lower memory usage in store-gateway and ingester**
+  We made various tweaks related to how index lookups are performend and how the active series customer trackers are implemented, resulting in better performance and lower overall memory usage in these two components.
+
+- **Improved Mimir dashboards**
+  We added two completely new dashboards named `Mimir / Overview resources` and `Mimir / Overview networking`, furthermore we have made various improvements to these existing dashboards:
+  - `Mimir / Overview`: Add "remote read", "metadata", and "exemplar" queries.
+  - `Mimir / Writes`: Add optional row about the distributor's new forwarding feature.
+  - `Mimir / Tenants`: Add insights into the read path.
+
+## Bug fixes
+
+- Flusher: Add `Overrides` as a dependency to prevent panics when starting with `-target=flusher`. [PR 3151](https://github.com/grafana/mimir/pull/3151)
```

Comment:
```suggestion
- Flusher: Added `Overrides` as a dependency to prevent panics when starting with `-target=flusher`. [PR 3151](https://github.com/grafana/mimir/pull/3151)
```